code/__HELPERS/unsorted.dm

```diff
@@ -1630,3 +1630,34 @@ Game Mode config tags:
 	if(!istype(T2))
 		T2 = get_turf(B)
 	return sqrt(((T2.x - T1.x) ** 2) + ((T2.y - T1.y) ** 2))
+
+// For surgeries
+
+
+/proc/find_working_surface_at_mob(mob/living/carbon/human/target, list/possible_surfaces_list)
+	for (var/surface in possible_surfaces_list)
+		world << "Trying to find: [surface]"
```

Comment:
You'll want to remove this once you're finished

Reply:
Oops